kubernetes/samples/charts/traefik/README.md

```diff
@@ -0,0 +1,73 @@
+# Install and Configure Traefik
+## Install Traefik Operator with Helm Chart
+Traefik helm chart is in official Charts of Helm: https://github.com/helm/charts/tree/master/stable/traefik. 
+The chart is in the default repository for Helm which is located at https://kubernetes-charts.storage.googleapis.com/ and is installed by default.
+
+To install Traefik operator to namespace `traefik` with default settings:
+```
+helm install --name traefik-operator --namespace traefik stable/traefik
+```
+Or with a given values.yaml:
+```
+helm install --name traefik-operator --namespace traefik --values values.yaml stable/traefik
+```
+With dashboard enabled, you can access the Traefik dashboard with URL `http://${HOSTNAME}:30305` with http Host `traefik.example.com`.
+```
+curl -H 'host: traefik.example.com' http://${HOSTNAME}:30305/
+```
+
+## Optionally Download Traefik Helm Chart
+You can download Traefik helm chart and untar it to a local folder if you want.
+```
+$ helm fetch  stable/traefik --untar
+```
+
+## Configure Traefik as Load Balancer for WLS Domains
+This chapter we'll demonstrate how to use Traefik to handle traffic to backend WLS domains.
+
+### 1. Install some WLS Domains
```

Comment:
Install WLS Domains
Now we need to prepare some backends for Traefik to do load balancing.

URL context

Note: After all the WLS domains are running, we need to stop the WLS operator and remove the per-domain Ingresses created by the WLS operator. Otherwise the WLS operator keeps monitoring the Ingresses and restores them to the original version if they are changed.